src/main/java/com/hm/achievement/utils/AchievementPlaceholderHook.java

```diff
@@ -25,6 +26,22 @@ public AchievementPlaceholderHook(AdvancedAchievements plugin) {
 
 	@Override
 	public String onPlaceholderRequest(Player p, String identifier) {
+
+		if (identifier.equalsIgnoreCase("achievements")) {
+			return String.valueOf(plugin.getCacheManager().getPlayerTotalAchievements(p.getUniqueId()));
+		}
+
+		if (identifier.equalsIgnoreCase("achievements_percentage")) {
+			return String.format("%.1f",
+					100 * (double) plugin.getCacheManager().getPlayerTotalAchievements(p.getUniqueId())
+							/ plugin.getAchievementsAndDisplayNames().size())
+					+ "%";
```

Comment:
Last suggestion before I merge. As you're using `String.format`, you could avoid the string concatenation `+ "%"` and include it inside the call: `String.format("%.1f%%", ...);`. Note the double `%%`, which allows to escape the percent sign. ^^

Reply:
Ok good! :)